lib/vdsm/gluster/cli.py

```diff
@@ -426,7 +426,10 @@ def _parseVolumeInfo(tree):
             value["volumeStatus"] = VolumeStatus.OFFLINE
         value['brickCount'] = el.find('brickCount').text
         value['distCount'] = el.find('distCount').text
-        value['stripeCount'] = el.find('stripeCount').text
+        if el.find('stripeCount') is not None:
+            value['stripeCount'] = el.find('stripeCount').text
+        else:
+            value['stripeCount'] = 1
```

Comment:
This adds an integer

I think a better way to do this is (did not test):

    value['stripeCount'] = el.findtext("stripeCount", "1")

https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.ElementTree.findtext

Reply:
Yes make sense it works i tested.

Reply:
Addressed